spec/index.html

```diff
@@ -7149,66 +7200,40 @@ <h5>STRAFTER</h5>
               <table>
                 <tbody>
                   <tr>
-                    <td>strafter("abc","b")</td>
-                    <td>"c"</td>
-                  </tr>
-                  <tr>
-                    <td>strafter("abc"@en,"ab")</td>
-                    <td>"c"@en</td>
-                  </tr>
-                  <tr>
-                    <td>strafter("abc"@en,"b"@cy)</td>
-                    <td>error</td>
-                  </tr>
-                  <tr>
-                    <td>strafter("abc"^^xsd:string,"")</td>
-                    <td>"abc"^^xsd:string</td>
+                    <td>`strAfter("abc","b")`</td>
+                    <td>`"c"</td>
                   </tr>
                   <tr>
-                    <td>strafter("abc","xyz")</td>
-                    <td>""</td>
+                    <td>`strAfter("abc"@en,"ab")`</td>
+                    <td>`"c"@en`</td>
                   </tr>
                   <tr>
-                    <td>strafter("abc"@en, "z"@en)</td>
-                    <td>""</td>
+                    <td>`strAfter("abc"@en,"b"@cy)`</td>
+                    <td><i>error</i></td>
                   </tr>
                   <tr>
-                    <td>strafter("abc"@en, "z")</td>
-                    <td>""</td>
+                    <td>`strAfter("abc"^^xsd:string,"")`</td>
+                    <td>`"abc"^^xsd:string`</td>
                   </tr>
                   <tr>
-                    <td>strafter("abc"@en, ""@en)</td>
-                    <td>"abc"@en</td>
+                    <td>`strAfter("abc","xyz")`</td>
+                    <td>`""`</td>
                   </tr>
                   <tr>
-                    <td>strafter("abc"@en, "")</td>
-                    <td>"abc"@en</td>
+                    <td>`strAfter("abc"@en, "z"@en)`</td>
+                    <td>`""`</td>
                   </tr>
-                </tbody>
-              </table>
-            </div>
-          </section>
-          <section id="func-encode">
```

Comment:
Move only: see [this comment](https://github.com/w3c/sparql-query/pull/184/files#r1927081549)